include/mbedtls/mbedtls_config.h

```diff
@@ -1650,6 +1650,18 @@
 */
 #define MBEDTLS_SSL_EARLY_DATA
 
+/**
+ * \def MBEDTLS_SSL_MAX_EARLY_DATA_SIZE
+ *
+ * The maximium amount of 0-RTT data. It only work when MBEDTLS_SSL_EARLY_DATA
+ * is enabled.
+ *
+ * This feature is experimental, not completed and thus not ready for
+ * production.
+ *
+ */
+#define MBEDTLS_SSL_MAX_EARLY_DATA_SIZE        1024
```

Comment:
Does 1024 come from the rfc8466?

Reply:
No, there are no limitation in RFC8446. That's just a reasonable value.

Reply:
Is it end?

Reply:
I am not sure 1024 is enough or not, can we only leave it here? If it is reasonable, I am file.